js/bookCollection.js

```diff
@@ -1,64 +1,65 @@
-const bookTitle = document.querySelector('.title');
-const bookAuthor = document.querySelector('.author');
-const AddButton = document.querySelector('.add');
-
-const books = [];
-
 const bookList = document.querySelector('.book-list');
+const BookTitle = document.querySelector('.title');
+const BookAuthor = document.querySelector('.author');
+const addButton = document.querySelector('.add');
 
```

Comment:
- [ ] [OPTIONAL]  Please, the value given to `Title` and `Author` inputs are not reflected in my case. Please fix this problem.

![image](https://user-images.githubusercontent.com/63915024/182665000-3fe1e699-40e1-4274-aa77-9a3a6bff381c.png)